src/workloads/query/AggregateExpressions.yml

```diff
@@ -1470,11 +1470,9 @@ AutoRun:
   - When:
       mongodb_setup:
         $eq:
-          - standalone
-          - standalone-80-feature-flags
-          - standalone-all-feature-flags
-          - standalone-classic-query-engine
-          - standalone-sbe
+          - replica-80-feature-flags
```

Comment:
I don't think we currently have any variants in sys-perf that run with `replica-80-feature-flags`

Reply:
I saw a bunch of tests with `replica-80-feature-flags` in the repo already. Do you know what's going on with those?

Reply:
I'm not sure - the only PR I could find related to it was https://github.com/10gen/dsi/pull/1637 but I couldn't find any that added/removed the variant and it wasn't removed as part of [SERVER-89499](https://jira.mongodb.org/browse/SERVER-89499). I can follow-up more on this if you think having a variant with the 8.0 FFs is necessary.

Reply:
I think we _do_ want an 8.0 FFs variant, since backports and such are still running hither and thither.

Reply:
Now that we've committed to releasing 8.0 with the feature flags we ended up choosing, having an 8.0 feature flags variant isn't that useful. However, I'd advocate to leave the specifications intact for now and have the Product Performance team do the work to wholesale remove these and ensure that we have good substitutes intact.

I've created a ticket to track that effort [here](https://jira.mongodb.org/browse/PERF-5857).

Reply:
Thanks @BlakeIsBlake for opening that ticket. As Blake said I'm happy for you to leave tests that have `replica-80-feature-flags` as is, and when our team gets to [PERF-5857](https://jira.mongodb.org/browse/PERF-5857) we can easily switch them to a 9.0/all FF variant. Just bear in mind, that it may be slightly confusing that the Genny AutoRuns are implying it's running on an 8.0 FF variant when it actually isn't.